core/db_models/EEM_WP_User.model.php

```diff
@@ -100,6 +100,16 @@ protected function __construct($timezone = null, ModelFieldFactory $model_field_
             'Venue'          => new EE_Has_Many_Relation(),
             'Message'        => new EE_Has_Many_Relation(),
         );
+        $this->foreign_key_aliases = [
+            'Event.EVT_wp_user'          => 'WP_User.ID',
+            'Payment_Method.PMD_wp_user' => 'WP_User.ID',
+            'Price.PRC_wp_user'          => 'WP_User.ID',
+            'Price_Type.PRT_wp_user'     => 'WP_User.ID',
+            'Question.QST_wp_user'       => 'WP_User.ID',
+            'Question_Group.QSG_wp_user' => 'WP_User.ID',
+            'Venue.TKT_wp_user'          => 'WP_User.ID',
+            'Ticket.VNU_wp_user'         => 'WP_User.ID',
```

Comment:
Should this be:

```suggestion
            'Ticket.TKT_wp_user'          => 'WP_User.ID',
            'Venue.VNU_wp_user'         => 'WP_User.ID',
```
?

Reply:
ya me likey, thnx will update